k9mail/src/main/java/com/fsck/k9/notification/AuthenticationErrorNotifications.java

```diff
@@ -59,8 +59,8 @@ public void clearAuthenticationErrorNotification(Account account, boolean incomi
 
     PendingIntent createContentIntent(Context context, Account account, boolean incoming) {
         Intent editServerSettingsIntent = incoming ?
-                AccountSetupIncoming.intentActionEditIncomingSettings(context, account) :
-                AccountSetupOutgoing.intentActionEditOutgoingSettings(context, account);
+                IncomingView.intentActionEditIncomingSettings(context, account) :
```

Comment:
When we setup a new account, the activity will finish when the whole process ends. But sometimes the activity is started for only one view in it. My opinion is using some flags to control it (sounds silly and not elegant). Could you please tell me whether there is some better way?